[PL-Doc] Move project across organizations (#11619)
harness/developer-hub: docs/platform/organizations-and-projects/move-projects/project-movement-checklist.md

---
title: Project movement checklist
description: Pre-move checklist and post-move  steps to ensure successful project movement across organizations.
sidebar_label: Project Movement Checklist
sidebar_position: 2
tags:
  - organizations
  - projects
  - project-movement
  - checklist
  - verification
keywords:
  - project move checklist
  - pre-move verification
  - post-move Remediation
---

The below checklist can be used for both pre-move verification and post-move remediation to ensure a project is moved with minimal disruption.

## Pre-move verification

When a project is moved, you may see warnings for unsupported entities or broken references. Key entities to watch are listed below:

1. Pipelines:
    - Pipelines that refer organization-level connectors will break after the move.
    - Secrets used in pipelines may be scoped at the organization level. After moving to a destination organization, these secrets might become unavailable. 
    - Templates used to build pipelines may be scoped at the organization level. Review and make these templates available in the destination organization; otherwise, pipelines may fail to render or execute.

>     - YAML entities may use fully qualified identifiers, such as orgIdentifiers, that reference the source organization. These references may become stale and cause errors.

Reviewer: ```suggestion
    - YAML entities may use fully qualified identifiers, such as `orgIdentifier`, that reference the source organization. These identifiers will become obsolete post move.
```

    - [Pipeline chaining](/docs/platform/pipelines/pipeline-chaining) will fail if the child pipeline’s project is moved.

>     - Pipelines that are running during the project move may fail.

Reviewer: ```suggestion
    - Pipelines that are running when a project is moved may fail.
```
